sound/effect.h

```diff
@@ -218,6 +218,10 @@ class Effect {
     return ret;
   }
 
+  size_t get_min_file() { return min_file_;}
```

Comment:
space after semicolon
This function should be "const": size_t get_min_file() const { return min_file_; }
Also applies to get_directory below.

Reply:
thanks, updated.